packages/grafana-ui/src/components/PanelMenu/PanelMenuItem.tsx

```diff
@@ -0,0 +1,108 @@
+import React from 'react';
+import { css } from '@emotion/css';
+
+import { GrafanaTheme2, PanelMenuItem } from '@grafana/data';
+import { Icon } from '../Icon/Icon';
+import { IconName } from '../../types';
+import { useStyles2 } from '../../themes';
+
+interface Props {
+  item: PanelMenuItem;
+  className?: string;
+  children?: React.ReactNode;
+}
+
+export const PanelMenuListItem = ({ item, className, children }: Props) => {
+  const styles = useStyles2(panelMenuItemStyles);
+  return (
+    <li className={styles.subMenuItemList} role="none">
+      <button
+        onClick={item.onClick}
+        role="menuitem"
+        tabIndex={-1}
+        className={styles.menuItem}
+        aria-keyshortcuts={item.shortcut}
+      >
+        <Icon name={item.iconClassName as IconName} className={styles.icon} aria-hidden />
+        <span className={styles.menuItemText}>{item.text}</span>
+        {item.shortcut && (
+          <div className={styles.shortcutContainer}>
+            <Icon name="keyboard" className={styles.shortcutIcon} aria-hidden />
+            <span className={styles.shortcutText} aria-hidden>
+              {item.shortcut}
+            </span>
+          </div>
+        )}
+        {children && <Icon name="angle-right" className={styles.angleRight} aria-hidden />}
+      </button>
+      {children}
```

Comment:
The `More...` menu item causes a bit of confusion. It's probably not that big of a deal, as the submenu items comes directly afterwards.